src/components/LoginForm.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+function LoginForm() {
+    return (
+    <div className="flex justify-center">
+
+        {/*Za manje ekrane*/}
```

Comment:
Making 2 forms for 2 different screen sizes doesn't make sense when using tailwind. It can all be solved with the use of min and max and max:disable min:disable.